### ISSUE TITLE
Pasting in cmd.exe

### ISSUE THREAD

**Issue by lipkau** [ewerybody/a2.modules#7]
AFAIK, Windows 10 now allows the user to use <kbd>Ctrl</kbd>+<kbd>V</kbd> in the cmd.exe prompt.
The current behavior of the module should be tested in this regard. And if so, the script should not bind the keys if WinVer >= Win10.

* This would be a great example for the script to update the UI, by disabling the Hotkey element

**Comment by ewerybody** [comment]
Yeah I already thought about something like this when porting CalculAid. The window classes there are diffenrently. But ok. This could even be something that every a2element supports: No only/not show up on certain OS versions.
